sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -335,6 +339,5 @@ def get_keyed_state(self, key):
     if not self.existing_keyed_state.get(key):
       self.existing_keyed_state[key] = DirectUnmergedState()
     if not self.partial_keyed_state.get(key):
-      self.partial_keyed_state[key] = (
-          self.existing_keyed_state[key].copy())
+      self.partial_keyed_state[key] = (self.existing_keyed_state[key].copy())
```

Comment:
<!--new_thread; commit:2bc3b2426c278088706b430164375d35dc5cd148; resolved:0-->
No need for extra parentheses.